pkg/crc/machine/machine.go

```diff
@@ -415,6 +415,11 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		return *result, errors.Newf("Error approving the node csr %v", err)
 	}
 
+	if proxyConfig.IsEnabled() {
+		logging.Info("Adding proxy details to cluster ...")
```

Comment:
"Waiting for the proxy configuration to be applied" or something like this? "Adding" sounds like we are doing an action on th ecluster when we are only waiting for things to happen :)

Reply:
@cfergeau since here we are already check for `IsEnabled()` does it make sense to check https://github.com/code-ready/crc/pull/1072/files#r388997725 here?

Reply:
We are adding a new exported `CheckProxyIsPartOfAuthOperator` method in the `cluster` package. It's currently only called from `machine.go`, but since it's part of a different package, maybe one day there will be more users calling it. Either we consider it's an error to call `CheckProxyIsPartOfAuthOperator` if `proxy.IsEnabled()` is false, in which case it would still be nice for `CheckProxyIsPartOfAuthOperator` to have a warning about it, or we consider it's ok to call `CheckProxyIsPartOfAuthOperator` if `proxy.IsEnabled()` is false, and then it should return true. In any case, I don't think we should rely on the rest of the code to have properly checked that proxy is enabled before calling that method.